docs/index.asciidoc

```diff
@@ -368,6 +369,14 @@ Number of items we can keep in the local queue before uploading them
 
 Specify how many workers to use to upload the files to S3
 
+[id="plugins-{type}s-{plugin}-upload_multipart_threshold"]
```

Comment:
Syntax is good, but would you mind moving this above `upload_queue_size` to maintain alphabetical order

Reply:
done